main.go

```diff
@@ -40,6 +42,8 @@ var proxyUpstreamURL = flag.String(
 	`Upstream HTTPS proxy where to forward traffic (e.g. "http://superproxy.com:8080")`,
 )
 
+var stickyStorage = flag.String("stickiness-db-path", proxy.MemoryStorage, "Path to the database for stickiness mapping")
```

Comment:
I suggest to rename stickyStorage -> stickyStoragePath as stickyStorage sounds more like an object.

Reply:
Done.